ci/cases/pr/C48_S2SWA_gefs.yaml

```diff
@@ -17,3 +17,5 @@ arguments:
   edate: 2021032312
   yaml: {{ HOMEgfs }}/ci/cases/yamls/gefs_ci_defaults.yaml
 
+skip_ci_on_hosts:
+  - wcoss2
```

Comment:
Assume this is because there is no dual-build on the shell system yet.

Reply:
Yes